go.mod

```diff
@@ -37,7 +36,7 @@ require (
 	github.com/oapi-codegen/runtime v1.1.1
 	github.com/operator-framework/api v0.27.0
 	github.com/operator-framework/operator-lifecycle-manager v0.27.0
-	github.com/percona/everest-operator v0.6.0-dev1.0.20241203113640-8dd4a9d32733
+	github.com/percona/everest-operator v0.6.0-dev1.0.20241220132035-6764e46d49e4
```

Comment:
Leaving a comment to not forget to update the value to the latest main when https://github.com/percona/everest-operator/pull/615 is merged